R/guide-legend.r

```diff
@@ -4,8 +4,8 @@
 #' Legend guides for various scales are integrated if possible.
 #'
 #' Guides can be specified in each `scale_*` or in [guides()].
-#' `guide="legend"` in `scale_*` is syntactic sugar for
-#' `guide=guide_legend()` (e.g. `scale_color_manual(guide = "legend")`).
+#' `guide = "legend"` in `scale_*` is syntactic sugar for
```

Comment:
re. my comment about backticks around arguments and their value, here there are backticks around both argument and value. I think we're wrapping both (see http://style.tidyverse.org/news.html#general-style), so I'd just move the backticks where mentioned earlier to encompass the argument name.

Reply:
I agree in principle however "default" (from above) is not an actual argument name where `guide` is. Thoughts?

Reply:
Oh, oh! I missed that. That makes sense. I almost think that it'd be better not to use `=`, in that case (e.g. is `0.5` by default, or default is `0.5` or whatever).